examples/gnrc_networking_subnets/setup_taps.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+PREFIX=2001:db8::/60
+
+SCRIPTPATH=$(dirname $(realpath "$0"))
+
+RIOTBASE=$SCRIPTPATH/../..
+RIOTTOOLS=$RIOTBASE/dist/tools
+TOPOLOGY=$SCRIPTPATH/tapology.txt
+
+setup() {
+    # add address to loopback so we can test reaching an
+    # address outside the test net
+    sudo ip a a fd00:dead:beef::1/128 dev lo
+
+    echo "creating tap interfaces"
+    i=0
+    sed '/^#/d' "$1" | while read -r level num; do
+        # we actually want to execute the output here.
+        # shellcheck disable=SC2091
+        $(printf -- "sudo %s/tapsetup/tapsetup -b br%s -t tap_%s -c %s\n" "$RIOTTOOLS" "$i" "$level" "$num") > /dev/null;
+        i=$((i+1))
+    done
+
+    # start radvd with a large prefix
+    sudo "$RIOTTOOLS"/radvd/radvd.sh -c br0 $PREFIX
```

Comment:
Would be nice if `PREFIX::1` could be added to `br0`, so one can test this by pinging the host system.
```suggestion
    sudo ip addr add ${PREFIX/::\//::1\/} dev br0
    # start radvd with a large prefix
    sudo "$RIOTTOOLS"/radvd/radvd.sh -c br0 $PREFIX
```

Reply:
The setup script will already add `fd00:dead:beef::1` to `lo` for that

Reply:
> The setup script will already add `fd00:dead:beef::1` to `lo` for that

But why not go with common convention and have the upstream gateway be `PREFIX::1`?